components/WalletLayout.vue

```diff
@@ -71,36 +67,8 @@ export default {
       if(!newVal) {
         this.showDrawer = false;
       }
-    },
-    async account(newVal) {
-      // FIXME: move this code into vanilla Web app layer (out of Vue layer)
-
-      // if not in CHAPI and the user logs in, ensure the credential handler
-      // gets installed
-      const accountId = newVal;
-      if(!this.$route.meta.chapi && accountId) {
-        try {
-          // install credential handler
-          await installHandler({url: '/credential-handler'});
-        } catch(e) {
-          // eslint-disable-line no-console
-          console.error('CHAPI register error:', e);
-          this.$q.notify({
-            type: 'negative',
-            message:
-              'Your wallet will not be shown to you as an option for using ' +
-              'credentials on other websites. Please reload the page and ' +
-              'select "Accept" to change this.',
-            actions: [{icon: 'fa fa-times'}]
-          });
-        }
-      }
     }
   },
-  beforeCreate() {
-    this._accountService = new AccountService();
-    this._tokenService = new TokenService();
```

Comment:
It doesn't look like these are used here anymore ... and I guess that's why you removed them? Presumably they aren't used by some other component in a hacky way that accesses them either, right? +1 to their removal if they are just leftovers that should have been removed before.

Reply:
Tested in the wallet and did not see them being used so hopefully not.